packages/flutter/test/material/dialog_test.dart

```diff
@@ -607,4 +607,46 @@ void main() {
     expect(find.text('0'), findsNothing);
     expect(find.text('1'), findsNothing);
   });
+
+  // Regression test for https://github.com/flutter/flutter/issues/28505.
+  testWidgets('showDialog only gets Theme from context on the first call', (WidgetTester tester) async {
+    Widget buildFrame(Key builderKey) {
+      return MaterialApp(
+        home: Material(
+          child: Center(
+            child: Builder(
+              key: builderKey,
+              builder: (BuildContext outerContext) {
+                return RaisedButton(
+                  onPressed: () {
+                    showDialog<void>(
+                      context: outerContext,
+                      builder: (BuildContext innerContext) {
```

Comment:
could be => ?

Reply:
This could be, but since `const AlertDialog(title: Text('Title'));` would have to be on it's own line anyways, I think its better to leave with `{ ... }`.